src/components/FoodPantryCard/index.tsx

```diff
@@ -12,11 +12,8 @@ const FoodPantryCard: FunctionComponent<Props> = ({ pantry }) => {
       <h1 id="pantry_address">
         Address: {pantry.address.streetNumber} {pantry.address.streetName}, {pantry.address.zipCode}
       </h1>
-      <h1 id="hours">
-        Hours: Monday - {pantry.hours.monday} Tuesday - {pantry.hours.tuesday} Wednesday - {pantry.hours.wednesday}{' '}
-        Thusday - {pantry.hours.thursday} Friday - {pantry.hours.friday} Saturday - {pantry.hours.saturday} Sunday -{' '}
-        {pantry.hours.sunday}
-      </h1>
+      <h1 id="pantry_email">Email: "{pantry.email}"</h1>
```

Comment:
Can remove these quotes
```suggestion
      <h1 id="pantry_email">Email: {pantry.email}</h1>
```